prowler/providers/aws/services/rds/rds_service.py

```diff
@@ -54,7 +54,7 @@ def __describe_db_instances__(self, regional_client):
                             self.db_instances.append(
                                 DBInstance(
                                     id=instance["DBInstanceIdentifier"],
-                                    endpoint=instance["Endpoint"]["Address"],
+                                    endpoint=instance.get("Endpoint"),
```

Comment:
Why are you retrieving the `Endpoint` dict if you need the `Address` in the check?

Reply:
Because endpoint is a dictionary that contains the address and the port.